addons/mail/static/src/core/common/thread_model.js

```diff
@@ -315,6 +315,12 @@ export class Thread extends Record {
             msg.thread = this;
         },
     });
+    /** @type {String|undefined} */
+    access_token;
+    /** @type {String|undefined} */
+    hash;
+    /** @type {integer|undefined} */
+    pid;
```

Comment:
I have no clue what `pid` is? Would be nice a short comment so I know what this is. Could be `partner id` or `project id`... or `parented id`? Idk ^^'